java/src/jmri/jmrit/beantable/LogixTableAction.java

```diff
@@ -98,12 +110,19 @@
  * lists:<br>
  * 1. the previous policy - Trigger on change of state only <br>
  * 2. the new default - Trigger on any enabled state calculation
- * <p>
  * Jan 15, 2011 - Pete Cressman
- *
+ * <p>
```

Comment:
Yes, add it to each method. Something like ```@since 4.7.3``` (if the class has a since tag, it will reflect when the class was added to JMRI). This way, when someone is using the documentation in an IDE (as shown in screenshot while typing in a method), the since tag would be with that documentation snippet.
<img width="598" alt="screen shot 2017-03-27 at 19 39 56" src="https://cloud.githubusercontent.com/assets/297232/24382625/66cc7338-1325-11e7-83db-f22ed254fc38.png">